config/vufind/Alma.ini

```diff
@@ -7,6 +7,7 @@ apiKey = "your-key-here"
 ; vufind    Use VuFind's user database for authentication -- patrons are retrieved
 ;           from Alma without a password (default)
 ; password  Use password authentication with Alma internal users
+; email     Username needs to be a valid email address for the user
```

Comment:
We should probably add some comments to config.ini as well at this point.

Reply:
Done.